Bob rewrite -- draft (#3)
kytrinyx/problem-specifications: exercises/bob/introduction.md

# Introduction

Bob is a teenager.

> He's not very smart, but he likes to think that he's very cool.

Reviewer: I wonder if the "He's not very smart" part adds anything. It feels somewhat judgemental?

Reviewer 2: Agreed.

```suggestion
He likes to think that he's very cool.
```

He definitely doesn't get excited about things.
That wouldn't be cool.

When people talk to him, his responses are pretty limited.

> If you ask him a question, such as "How are you?", Bob answers 'Sure.'

Reviewer: Was the single/double quote inconsistency intended?

```suggestion
If you ask him a question, such as "How are you?", Bob answers "Sure."
```

Author: Not intended! Nice catch, thank you.

He answers 'Whoa, chill out!' if you YELL AT HIM (in all capitals).

He answers 'Calm down, I know what I'm doing!' if you yell a question at him.

He says 'Fine. Be that way!' if you address him without actually saying anything.

> He answers 'Whatever.' to anything else.

Reviewer: Not saying this would be better, but did you consider making the five different types of responses into a list?

# Introduction

Bob is a teenager.
He's not very smart, but he likes to think that he's very cool.

> He definitely doesn't get excited about things.

Reviewer 2: ```suggestion
And he definitely doesn't get excited about things.
```

That wouldn't be cool.

When people talk to him, his responses are pretty limited.
In fact, he only ever answers one of five things:

- _"Sure."_
  This is his response if you ask him a question, such as "How are you?"
- _"Whoa, chill out!"_
  This is his answer if you YELL AT HIM (in all capitals).
- _"Calm down, I know what I'm doing!"_
  This is what he says if you yell a question at him.
- _"Fine. Be that way!"_
  This is how he responds if you address him without actually saying anything.
- _"Whatever."_

>   This is what he answers to anything else.

Reviewer: Personally, I like this version better! It makes it more clear that there are exactly five response and their definitions are now grouped together, which I like.

Reviewer 2: I have a preference for bold (`**`) not italic (`_`) formatting as I feel it makes it stand out more on the website. I think italics things tend to blur into the background more, so despite `_` being for emphasis, I actually think it has the opposite UX effect when rendered. So I'd change these to bold.

# Introduction

> Bob is a teenager.

Reviewer 2: ```suggestion
Bob is a [lackadaisical](https://www.collinsdictionary.com/dictionary/english/lackadaisical) teenager.
```

This needs to use the non-inline markdown syntax though.
